sourcecode.md

```diff
@@ -10,7 +10,9 @@
 	- device - It contains files specific to each vendor device. In general, it contains the python scripts for accessing EEPROM, SFP, PSU, LED,etc., specific to the device hardware.
 	- dockers - This folder contains sub-folders for all dockers running in the SONiC. Each of those sub-folders contains files that explains about the processes that need to run inside that docker. List of dockers and the processes running inside each docker is given at the end of this document.
 	- files - Contain multiple sub-folders required for building and running SONiC services.   
-		(a) Aboot, (b) apt - few default files related to  for "apt-get" ,"apt-*" applications (c) build_templates - Contains the jinja2 template files to generate (as part of "build process") the systemd services files required for starting the various dockers. It also contains the file sonic_debian_extension.j2 is used in "build process"; it copies the required files and installs the required packages in the "/fsroot" which is built ass part of the SONiC image.  
+		(a) Aboot,
+		(b) apt - few default files related to  for "apt-get" ,"apt-*" applications 
+		(c) build_templates - Contains the jinja2 template files to generate (as part of "build process") the systemd services files required for starting the various dockers. It also contains the file sonic_debian_extension.j2 is used in "build process"; it copies the required files and installs the required packages in the "/fsroot" which is built ass part of the SONiC image.  

```

Comment:
While you're here you might want to fix the typo "ass" -> "as". I was about to raise a PR myself when I found yours.